integrations/security/OCSF_1_1_0/readme.md

```diff
@@ -0,0 +1,100 @@
+    - OCSF 3001 - Account Change
+    - OCSF 3002 - Authentication
+    - OCSF 3003 - Authorize Session
+    - OCSF 3004 - Entity Management
+    - OCSF 3005 - User Access Management
+    - OCSF 3006 - Group Management
+  - Network Activity
+    - OCSF 4001 - Network Activity
+    - OCSF 4002 - HTTP Activity
+    - OCSF 4003 - DNS Activity
+    - OCSF 4004 - DHCP Activity
+    - OCSF 4005 - RDP Activity
+    - OCSF 4006 - SMB Activity
+    - OCSF 4007 - SSH Activity
+    - OCSF 4008 - FTP Activity
+    - OCSF 4009 - Email Activity
+    - OCSF 4010 - Network File Activity
+    - OCSF 4011 - Email File Activity
+    - OCSF 4012 - Email URL Activity
+    - OCSF 4013 - NTP Activity
+  - Discovery 
+    - OCSF 5001 - Device Inventory Info
+    - OCSF 5002 - Device Config State 
+    - OCSF 5003 - User Inventory Info
+    - OCSF 5004 - Operating System Patch State
+    - OCSF 5019 - Device Config State Change 
+  - Application Activity
+    - OCSF 6001 - Web Resources Activity
+    - OCSF 6002 - Application Lifecycle
+    - OCSF 6003 - API Activity
+    - OCSF 6004 - Web Resources Access Activity
+    - OCSF 6005 - Datastore Activity
+    - OCSF 6006 - File Hosting Activity
+    - OCSF 6007 - Scan Activity
+
+### OpenSearch Ingestion template
+The OpenSearch Ingestion template (`assets/OSI-pipeline.yaml`) provides a template you can use with an OpenSearch Ingestion pipeline to ingest OCSF data. 
+
+### Index State Management (ISM) policy
+The ISM policy (`assets/ISM.json`) rollsover the indexes daily or when they have reached 40GB. The ISM policy also deletes indexes that are more than 15 days old.
+
+### Initialization scripts
+The initialization scripts helps set up the component templates, index templates, ISM policy, and aliases in the OpenSearch cluster. 
+
+There are two scripts - one that uses basic auth (`assets/os_init_basic_auth.py`) - and one that uses IAM auth (`assets/os_init_IAM_auth.py`). 
+
+### OpenSearch objects
+The OpenSearch objects (`assets/OCSF_objects.ndjson`) contains visualizations, dashboards, and index patterns to help you get started with exploring OCSF data. Visualizations include: 
+
+* OCSF High level overview (All OCSF categories) page
+![OpenSearch Dashboard](static/all_ocsf_overview.png)
+
+* OCSF Findings (2000 series) overview page
+![OpenSearch Dashboard](static/ocsf_findings_overview_2000_series.png)
+
+* Network Activity (4001) Org level overview
+![OpenSearch Dashboard](static/ocsf_4001_overview.png)
+
+* Network Activity (4001) Account Level Drill Down
+![OpenSearch Dashboard](static/ocsf_4001_drilldown.png)
+
+* DNS Activity (4003) Org level overview
+![OpenSearch Dashboard](static/ocsf_4003_overview.png)
+
+## Installation instructions
+1. Download the index and component template zip files. Upload it to an S3 bucket or save it to your local machine.
+2. Download the right initialization script based on how you would like to authenticate to OpenSearch (basic auth or AWS IAM). 
```

Comment:
plz give an example bash here
```
% cp {initialization script}
% edit {variables in the initialization script}
% ./script.sh
```

Reply:
one script covers all OCSF 1.1.0 standard...since this solution hinges on component templates, there are a good portion of those component templates that are shared across ALL categories...so for now, in this version, it goes as one complete package regardless if a set of class_uids are not used.  If we segregate by class_uid, were screwed because each commit that has a shared component template would need to traverse the dependencies in other source check-insif those are misaligned, it breaks others.  That would be something we could filter and de-select if the Integrations package was a bit more flexible than the tunnel vision approach it has now...one source, multiple impls on data source side, similar viziz

Reply:
The script is a bit complex to have it in bash. I'll add clearer instructions on where to modify the variables and to run it in an IDE

Reply:
ok cool
@Kevlw-AWS 
I would also love to see this as a packaged docker compose solution that anyone can just run and see a live containerized example in its local env